common/src/chain/config/mod.rs

```diff
@@ -454,10 +455,9 @@ impl ChainConfig {
         self.empty_consensus_reward_maturity_distance
     }
 
-    // TODO: this should be part of net-upgrades. There should be no canonical definition of PoW for any chain config
```

Comment:
This TODO is still valid: PoWChainConfig should not be a part of ChainConfig but rather net-upgrades so that we could change the parameters.